meta-cube/recipes-support/overc-utils/source/cube-ctl

```diff
@@ -894,6 +894,8 @@ case "${cmd}" in
 
 		${SBINDIR}/cube-cfg mount bind:/opt/container:/opt/container
 		${SBINDIR}/cube-cfg mount bind:/var/lib/cube:/var/lib/cube
+		${SBINDIR}/cube-cfg mount bind:/:/essential
+		${SBINDIR}/cube-cfg mount bind:/boot:/boot
```

Comment:
We don't need these bind mounts. It is too much coupling (and attack surface) in dom0 if they are present.

We can always escape to the parent namespace from dom0, and anything that was using these mounts previously should do that now.